src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java

```diff
@@ -1054,6 +1064,42 @@ public void createFile(
     }
   }
 
+  @StarlarkMethod(
+      name = "getenv",
```

Comment:
Hm, I'm used to seeing `environ` as a _collection_ of environment variables (or just keys, such as in `repository_rule.environ`). And, considering Starlark's Python roots, as an accessor method for a single key I figured users would be familiar w/ `getenv` as an analog of Python's `os.getenv`. That said, I'm not wedded to this, so I can go either way. (Or, since you suggest dropping `get`, maybe `rctx.env`?) WDYT?

Reply:
I also like `getenv` in analogy to Python's (and C's) `getenv` - we even already have `rctx.readdir`, which follows a similar naming scheme. With `rctx.env`, it's not so clear given just that the name that this is indeed a method and not just an alias of `rctx.os.environ`.

Reply:
OK, I'm convinced!